lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -48,7 +51,8 @@ export type AuthOptions = {
 };
 
 /**
- * Telemetry Config Options
+ * Use this to configure the telemetry options in the Configuration object
+ *
  * - applicationName              - Name of the consuming apps application
  * - applicationVersion           - Version of the consuming application
  * - telemetryEmitter             - Function where telemetry events are flushed to
```

Comment:
Remove this as it's not part of the options

Reply:
Done.